scst-store/create_service_account_access_token.md

```diff
@@ -21,16 +21,6 @@ To create a read-only service account, run the following command. The command cr
 ```sh
 kubectl apply -f - -o yaml << EOF
 apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
```

Comment:
I think the `Creating Service Accounts` section needs to be tweaked more.

The instruction for the RW service account is for creating and binding.  However, the instructions for the RO service account is only for binding (no creation).  We don't show instructions on how to create a RO service account.

Maybe the main section should be called `Service Accounts`.  Also, update the intro text of the RO service account  to mention binding to the account, and not creating it (as the app creates it for them).

Reply:
How does this update look?